recipes/log.c/all/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(log.c
+	    VERSION 0.1.0
+	    DESCRIPTION "A simple logging library implemented in C99"
+	    HOMEPAGE_URL "https://github.com/rxi/log.c"
+	    LANGUAGES C
+	)
+
+# Init Conan stuff
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+# Defines the CMAKE_INSTALL_LIBDIR, CMAKE_INSTALL_BINDIR and many other useful macros.
+# See https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
+include(GNUInstallDirs)
+
+set(SRC_FOLDER "source_subfolder")
```

Comment:
You can hardcore this path in the other lines 
It won't change.

Reply:
It's right after GNU paths sets